wire-java-generator/src/test/java/com/squareup/wire/java/JavaGeneratorTest.java

```diff
@@ -444,4 +444,19 @@ public void enclosingTypeIsNotMessage() {
             + "  }")
         .contains("public static final class B extends Message<B, B.Builder> {");
   }
+
+  @Test
+  public void generateTypeUsesPackageNameOnFieldAndClassNameClash() throws Exception {
+    RepoBuilder repoBuilder = new RepoBuilder()
+	    .add("person.proto", ""
+	        + "package common.proto;\n"
+	        + "enum Gender {\n"
+	        + "Gender_Male = 0;\n"
+	        + "Gender_Female = 1;\n"
+	        + "}\n"
+	        + "message Person {\n"
+	        + "optional Gender Gender = 1;\n"
+	        + "}\n");
+	    assertThat(repoBuilder.generateCode("common.proto.Person").contains("public final Gender common_proto_Gender;"));
```

Comment:
The assertion is wrong.
`assertThat(repoBuilder.generateCode("common.proto.Person").contains("public final Gender common_proto_Gender;"));`
should be
`assertThat(repoBuilder.generateCode("common.proto.Person")).contains("public final Gender common_proto_Gender;");`
you could have noticed it via the IDE warning, or by running the test without your fix in JavaGenerator.

also, add a new line before `.contains` for we authorize up to 100 char / line.